### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-ru/content_questlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#292]
```
translated questlist_v068, some changes in questlist.
```

Original issue reported on code.google.com by `e.solodo...@gmail.com` on 1 Jan 2012 at 11:47

Attachments:
- [content_questlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-292/comment-0/content_questlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
IMHO
It's need to use transcribed names of cities
and left "Vil'gard" instead "Podlograd"

```

Original comment by `shell.an...@gmail.com` on 2 Jan 2012 at 2:49

**Comment by GoogleCodeExporter** [comment]
```
Perhaps we shall create a thread "Translation in russian" in the game forum to 
unify all names?

```

Original comment by `shell.an...@gmail.com` on 2 Jan 2012 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
may be, you are right. but in previous version of translate was both translated 
and transliterated names. we just should define a standart for nomination.
i'll rework text; i think,  in the future we can meet more difficult names for 
translation. 
idea about thread is not bad, if some people will use it.
```

Original comment by `e.solodo...@gmail.com` on 2 Jan 2012 at 6:43

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:47
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
